test/reactTags.test.js

```diff
@@ -245,6 +245,29 @@ describe('Test ReactTags', () => {
     $input.simulate('keyDown', { keyCode: DOWN_ARROW_KEY_CODE });
   });
 
+  test('disables the input with disabled prop', () => {
```

Comment:
`should disable the input when disabled prop is true`

Reply:
can you actually put all the tests with the `disabled` prop inside a `describe` block ?
`describe('tests disabled prop')`